migrations/versions/e3784181a957_add_stripe_tables.py

```diff
@@ -0,0 +1,157 @@
+        )
+
+    with op.batch_alter_table("users", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("tier_id", sa.Integer(), nullable=True))
+        batch_op.add_column(sa.Column("stripe_customer_id", sa.String(length=255), nullable=True))
+        batch_op.add_column(
+            sa.Column("stripe_subscription_id", sa.String(length=255), nullable=True)
+        )
+        batch_op.add_column(
+            sa.Column("stripe_subscription_cancel_at_period_end", sa.Boolean, nullable=True)
+        )
+        batch_op.add_column(
+            sa.Column(
+                "stripe_subscription_status",
+                sa.Enum(
+                    "INCOMPLETE",
+                    "INCOMPLETE_EXPIRED",
+                    "TRIALING",
+                    "ACTIVE",
+                    "PAST_DUE",
+                    "CANCELED",
+                    "UNPAID",
+                    "PAUSED",
+                    name="stripesubscriptionstatusenum",
+                ),
+                nullable=True,
+            ),
+        )
+        batch_op.add_column(
+            sa.Column(
+                "stripe_subscription_current_period_end",
+                sa.DateTime(),
+                nullable=True,
+            ),
+        )
+        batch_op.add_column(
+            sa.Column(
+                "stripe_subscription_current_period_start",
+                sa.DateTime(),
+                nullable=True,
+            ),
+        )
+        batch_op.create_index("idx_users_stripe_customer_id", ["stripe_customer_id"], unique=False)
+        batch_op.create_foreign_key(None, "tiers", ["tier_id"], ["id"])
+
+    # Add the default tiers
+    tiers_table = sa.table(
+        "tiers",
+        sa.column("id", sa.Integer),
+        sa.column("name", sa.String),
+        sa.column("monthly_amount", sa.Integer),
+    )
+    op.bulk_insert(
+        tiers_table,
+        [
+            {"id": 1, "name": "Free", "monthly_amount": 0},
+            {"id": 2, "name": "Business", "monthly_amount": 2000},
+        ],
+    )
+
+
+def downgrade():
+    with op.batch_alter_table("users", schema=None) as batch_op:
+        batch_op.drop_constraint(None, type_="foreignkey")
+        batch_op.drop_index("idx_users_stripe_customer_id")
+        batch_op.drop_column("stripe_subscription_current_period_start")
+        batch_op.drop_column("stripe_subscription_current_period_end")
+        batch_op.drop_column("stripe_subscription_status")
+        batch_op.drop_column("stripe_subscription_cancel_at_period_end")
+        batch_op.drop_column("stripe_subscription_id")
+        batch_op.drop_column("stripe_customer_id")
+        batch_op.drop_column("tier_id")
+
+    with op.batch_alter_table("stripe_invoices", schema=None) as batch_op:
+        batch_op.drop_index(batch_op.f("ix_stripe_invoices_invoice_id"))
+
+    op.drop_table("stripe_invoices")
+    op.drop_table("tiers")
+    with op.batch_alter_table("stripe_events", schema=None) as batch_op:
```

Comment:
Don't need the explicit drop on the index since we drop the table in the next line.

Reply:
Fixed in 8984e5765a1295c8a4f7fb4dfa038b4563ed43c7